### PR TITLE
Adapted component to accept React Router links. Refactor.

### PR DESCRIPTION
**Pull request by kikoruiz**
# PR summary: #
* Adapted to accept React Router links.
* Modified docs to enable `pushState`.
* Enable lazy load optionally.
* Small refactor.

@davecarter @miduga @carlosvillu @ruben-martin-lozano please review!

### CODE REVIEW

**Comment by FranDelaCasa** [comment]
👍

File: src/sui-multimedia-image/index.jsx [hunk 1]
```diff
@@ -1,60 +1,60 @@
-import React from 'react';
+import React, { Component, PropTypes } from 'react';
 
-class SuiMultimediaImage extends React.Component {
+const DEFAULT_SRC = 'data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/PjwhRE9DVFlQRSBzdmcgIFBVQkxJQyAnLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4nICAnaHR0cDovL3d3dy53My5vcmcvR3JhcGhpY3MvU1ZHLzEuMS9EVEQvc3ZnMTEuZHRkJz48c3ZnIGhlaWdodD0iMzJweCIgc3R5bGU9ImVuYWJsZS1iYWNrZ3JvdW5kOm5ldyAwIDAgMzIgMzI7IiB2ZXJzaW9uPSIxLjEiIHZpZXdCb3g9IjAgMCAzMiAzMiIgd2lkdGg9IjMycHgiIHhtbDpzcGFjZT0icHJlc2VydmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiPjxnIGlkPSJMYXllcl8xIi8+PGcgaWQ9InhfeDVGX2FsdCI+PHBhdGggZD0iTTE2LDBDNy4xNjQsMCwwLDcuMTY0LDAsMTZzNy4xNjQsMTYsMTYsMTZzMTYtNy4xNjQsMTYtMTZTMjQuODM2LDAsMTYsMHogTTIzLjkxNCwyMS4wODYgICBsLTIuODI4LDIuODI4TDE2LDE4LjgyOGwtNS4wODYsNS4wODZsLTIuODI4LTIuODI4TDEzLjE3MiwxNmwtNS4wODYtNS4wODZsMi44MjgtMi44MjhMMTYsMTMuMTcybDUuMDg2LTUuMDg2bDIuODI4LDIuODI4ICAgTDE4LjgyOCwxNkwyMy45MTQsMjEuMDg2eiIgc3R5bGU9ImZpbGw6IzRFNEU1MDsiLz48L2c+PC9zdmc+';
```
Comment (miduga) [inline src/sui-multimedia-image/index.jsx]: O_O

**Comment by davecarter** [comment]
Genial Kiko!
👍

File: src/sui-multimedia-lazy-load/index.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import React, { Component, PropTypes } from 'react';
+import Spinner from '@schibstedspain/sui-spinner';
+import LazyLoad from 'react-lazy-load';
+
+export default class SuiMultimediaLazyLoad extends Component {
```
Comment (miduga) [inline src/sui-multimedia-lazy-load/index.jsx]: <3 esta está perfecta para hacerse stateless.

**Comment by miduga** [comment]
Muy rica! 👍

File: src/sui-multimedia/index.jsx [hunk 3]
```diff
@@ -1,60 +1,60 @@
...
+        );
+
+        const imageItem = lazyLoad ? multimediaLazyLoad : multimediaImage;
+
+        const imageWrapper = image.link ? (
+          <a key={index} href={image.link} title={image.alt}>
+            {imageItem}
+          </a>
+        ) : image.routerLink ? (
+            <Link key={index} to={image.routerLink}>
```
Comment (ruben-martin-lozano) [inline src/sui-multimedia/index.jsx]: Puedes meterle tambien el `title` como el enlace normal

**Pull request merged**